### PR TITLE
Added more logs for eventBatch execution error scenario

### PR DESCRIPTION
**Pull request by priyanshi-yb**

### CODE REVIEW

File: yb-voyager/cmd/live_migration.go [hunk 1]
```diff
@@ -318,7 +318,12 @@ func processEvents(chanNo int, evChan chan *tgtdb.Event, lastAppliedVsn int64, d
 			} else if tdb.IsNonRetryableCopyError(err) {
 				break
 			}
-			log.Warnf("retriable error executing batch on channel %v (last VSN: %d): %v", chanNo, eventBatch.GetLastVsn(), err)
+			log.Warnf("retriable error executing batch on channel %v (batch id: %s): %v", chanNo, eventBatch.GetEventID(), err)
+			vsns := make([]int64, 0, len(batch))
+			for _, e := range batch {
+				vsns = append(vsns, e.Vsn)
+			}
+			log.Warnf("vsns of events in the batch id (%s): %v", eventBatch.GetEventID(), vsns)
```
Comment (makalaaneesh) [inline yb-voyager/cmd/live_migration.go:326]: Can you combine this with the above log? would be easy to grep

File: yb-voyager/src/tgtdb/event.go [hunk 2]
```diff
@@ -361,6 +361,10 @@ func (eb *EventBatch) GetChannelMetadataUpdateQuery(migrationUUID uuid.UUID) str
 		migrationUUID, eb.ChanNo)
 }
 
+func (eb *EventBatch) GetEventID() string {
```
Comment (amit-yb) [inline yb-voyager/src/tgtdb/event.go:364]: Nit: name this method as just `ID()`. So the call would look like `eventBatch.ID()`.

**Review by amit-yb — COMMENTED** [review]
Will send remaining comments in Slack discussion thread.

File: yb-voyager/src/tgtdb/yugabytedb.go [hunk 3]
```diff
@@ -545,8 +545,12 @@ func (yb *TargetYugabyteDB) ExecuteBatch(migrationUUID uuid.UUID, batch *EventBa
 		if err != nil {
 			return false, fmt.Errorf("error creating tx: %w", err)
 		}
-		defer tx.Rollback(ctx)
-
+		defer func() {
+			err = tx.Rollback(ctx)
```
Comment (amit-yb) [inline yb-voyager/src/tgtdb/yugabytedb.go:549]: BUG: If a `Rollback()` is successful, this code will overwrite the original `err` and return success to the caller (see the named return value of the func passed to `WithConn()`). Use `errRollback := tx.Rollback(ctx)`.

**Comment by priyanshi-yb** [comment]
Taken some of these logs in this PR - https://github.com/yugabyte/yb-voyager/pull/1452

**Pull request closed without merging**